website/docs/r/automation_actions_action.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_automation_actions_action"
+sidebar_current: "docs-pagerduty-resource-automation-actions-action"
+description: |-
+  Creates and manages an Automation Actions action in PagerDuty.
+---
+
+# pagerduty\_automation\_actions\_action
+
+An Automation Actions [action](https://developer.pagerduty.com/api-reference/d64584a4371d3-create-an-automation-action) invokes jobs and workflows that are staged in Runbook Automation or Process Automation. It may also execute a command line script run by a Process Automation runner installed in your infrastructure.
+
+## Example Usage
+
+```hcl
+resource "pagerduty_automation_actions_action" "example" {
+  name = "PA Action created via TF"
+  description = "Description of the PA Action created via TF"
+  action_type = "process_automation"
+  action_data_reference {
+    process_automation_job_id = "P123456"
+  }
+}
```

Comment:
It would be valuable to have an additional example for actions of type **script**, don't you think?